### PR TITLE
Pivot: Fluent Updates

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [ ] Addresses an existing issue: #7009
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Updated the pivot's text color on pressed hover state to match current Fabric design spec in toolkit

**Before**
pivot-before

**After**
pivot-after

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9595)

### CODE REVIEW

**Comment by aftab-hassan** [comment]
@betrue-final-final, could you please help review this PR.

Also, looks like @jdhuntington is out this week.

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts [hunk 1]
```diff
@@ -59,7 +59,8 @@ const linkStyles = (props: IPivotStyleProps): IStyle[] => {
           cursor: 'pointer'
         },
         ':active': {
-          backgroundColor: palette.neutralLight
+          backgroundColor: palette.neutralLight,
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts]: is there a semantic slot we could use?

@phkuo @aneeshack4 fyi

Comment (phkuo) [reply packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts]: Maybe the slots for actionLinks/Buttons? The borderless, backgroundless buttons.

File: packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts [hunk 2]
```diff
@@ -54,12 +54,13 @@ const linkStyles = (props: IPivotStyleProps): IStyle[] => {
           visibility: 'hidden'
         },
         ':hover': {
-          backgroundColor: palette.neutralLighter,
+          backgroundColor: semanticColors.buttonBackground,
           color: semanticColors.actionLinkHovered,
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts]: text color and background colors should be pulling from the same "bucket"

**Comment by aftab-hassan** [comment]
@betrue-final-final, could you please take a look at this ?

**Pull request merged**